llvm/lib/Transforms/Scalar/LowerMatrixIntrinsics.cpp

```diff
@@ -1430,6 +1558,10 @@ class LowerMatrixIntrinsics {
       if (ShapeMap.contains(U.getUser()))
         continue;
 
+      if (auto *Intr = dyn_cast<IntrinsicInst>(U.getUser()))
+        if (tryLowerIntrinsic(Intr))
```

Comment:
This is done here because we don't have shape info for those. Wondering if it would be possible/cleaner to set shape info for them to 1x1?

Reply:
ooh nice idea. that works out nicely.